.circleci/config.yml

```diff
@@ -1441,7 +1462,17 @@ workflows:
           requires:
             - contracts-bedrock-build
       - contracts-bedrock-tests-upgrade:
-          name: contracts-bedrock-tests-upgrade
+          name: contracts-bedrock-tests-upgrade op-mainnet
+          fork_op_chain: op
+          fork_base_chain: mainnet
+          fork_base_rpc: https://ci-mainnet-l1-archive.optimism.io
+          fork_block_number: 21387830
```

Comment:
Is it possible to either: 
1. get this number from the justfile
2. remove it from the justfile and store it here instead
? 

Duplicating it in two places is sure to lead to confusion in the future.

Reply:
Yes. I was going to put this in another comment but got distracted. 

In fact we should update these numbers to something that comes after holocene, and within this PR is probably a good time to do it.

Reply:
Keeping in mind that we will be using this for a variety of chains and I would not like to have a mess in the just file maybe having the source of truth in the CI would be better?